README.md

```diff
@@ -1,177 +1,178 @@
-to the root of the workspace (which gets reset to the repository if you use the `checkout` action).
+With a different severity level:
 
-For example, to include a custom policy as: .anchore/policy.json in your code repository, set:
 ```yaml
- - uses: anchore/scan-action@v1
-       with:
-         image-reference: "localbuild/testimage:latest"
-         dockerfile-path: "Dockerfile"
-         fail-build: true
-         custom-policy-path: .anchore/policy.json
+ - name: Scan image
+   uses: anchore/scan-action@v2
+   with:
+     image: "localbuild/testimage:latest"
+     fail-build: true
+     severity-cutoff: Critical
 ```
 
-For an overview of policy format and the checks it can perform, see the [Anchore policy bundle documentation](https://docs.anchore.com/current/docs/engine/general/concepts/policy/bundles/)
 
 ### Action Inputs
 
-| Input Name | Description | Required | Default Value |
-|-----------------|-------------|----------|---------------|
-| image-reference | The image to scan | :heavy_check_mark: | N/A |
-| dockerfile-path | Path to a dockerfile used to build the image-reference image to add metadata for policy evaluation |  | null |
-| debug | Verbose logging output |  | false |
-| fail-build | Fail the build if policy evaluation returns a fail | | false |
-| include-app-packages | Include application packages for vulnerability matches. Requires more vuln data and thus scan will be slower but better results | | false |
-| custom-policy-path | A path to a policy json file for specifying a policy other than the default, which fails on >high vulnerabilities with fixes | | null |
-| anchore-version | An optional parameter to specify a specific version of anchore to use for the scan. Default is the version locked to the scan-action release | false | v0.7.3 |
-| acs-report-enable | Optionally, enable feature that causes a result.sarif report to be generated after successful action execution.  This report is compatible with GitHub Automated Code Scanning (ACS), as the artifact to upload for display as a Code Scanning Alert report. | | false |
-| acs-report-severity-cutoff | With ACS reporting enabled, optionally specify the minimum vulnerability severity to trigger an "error" level ACS result.  Valid choices are "Negligible", "Low", "Medium", "High" and "Critical".  Any vulnerability with a severity less than this value will lead to a "warning" result.  Default is "Medium". | | Medium |
-
-### Action Outputs 
-
-| Output Name | Description | Type | 
+The only required key is `image`; all the other keys are optional. These are all the available keys to configure this action, along with its defaults:
+
+| Input Name | Description | Default Value |
+|-----------------|-------------|---------------|
+| `image` | The image to scan | N/A |
+| `debug` | Verbose logging output | `false` |
+| `fail-build` | Fail the build if policy evaluation returns a fail | `false` |
```

Comment:
@alfredodeza - is the 'fail-build' parameter still supported (I could see the renamed severity-cutoff no being used instead of policy evaulation, to optionally instruct the action to exit if any findings exceed the cutoff, as opposed to the prior behavior of failing if a policy fail status was reported).  If so, then some of the help/verbiage should change when describing this parameter

Reply:
`fail-build` is still supported. I've updated the documentation to reflect the fact that this is not tied to policy evaluations, but rather, to severity